src/app/item-popup/ItemStat.tsx

```diff
@@ -17,28 +17,43 @@ export default function ItemStat({ stat, item }: { stat: DimStat; item: DimItem
   const masterworkValue =
     (item.isDestiny2() && item.masterworkInfo && item.masterworkInfo.statValue) || 0;
 
+  const modSocket = modSocketFor(item);
+  const moddedStatValue =
+    (item.isDestiny2() &&
+      modSocket &&
```

Comment:
```
import idx from 'idx';

idx(modSocket, (modSocket) => modSocket.plug.stats[stat.statHash]) || 0;

```

Reply:
nice! was hoping there was a better way to handle these nested checks...